src/Storages/ObjectStorage/StorageObjectStorageStableTaskDistributor.h

```diff
@@ -34,6 +39,7 @@ class StorageObjectStorageStableTaskDistributor
     std::unordered_set<String> unprocessed_files;
 
     std::vector<std::string> ids_of_nodes;
+    std::unordered_map<size_t, std::list<String>> processed_files;
```

Comment:
replica_to_files_to_be_processed_map or something like this.

imo

1. It needs to mention it is a mapping from replica to list of files
2. it can't be "processed" as it is in the past tense because the files haven't been processed yet as far as I understand